release/airflow/airflow_scripts.sh

```diff
@@ -0,0 +1,213 @@
+    WAIT_FOR_RESULT="true"
+
+    # uses the environment variables from next line + $PROJECT_ID $SVC_ACCT
+    create_subs_file "BRANCH" "CHECK_GREEN_SHA_AGE" "COMMIT" "GCS_RELEASE_TOOLS_PATH" "VERIFY_CONSISTENCY"
+    cat "${SUBS_FILE}"
+
+    run_build "cloud_get_commit.template.json" \
+         "${SUBS_FILE}" "${PROJECT_ID}" "${SVC_ACCT}" "${KEY_FILE_PATH}" "${WAIT_FOR_RESULT}"
+    exit "${BUILD_FAILED}"
+}
+
+function build_template() {
+    local KEY_FILE_PATH
+    local WAIT_FOR_RESULT
+    KEY_FILE_PATH=""
+    WAIT_FOR_RESULT="true"
+
+    # shellcheck disable=SC2034
+    GCR_PATH="${GCR_STAGING_DEST}"
+    GCS_PATH="${GCS_BUILD_PATH}"
+    VER_STRING="${VERSION}"
+    create_subs_file "BRANCH" "GCR_PATH" "GCS_PATH" "GCS_RELEASE_TOOLS_PATH" "VER_STRING"
+    cat "${SUBS_FILE}"
+
+    run_build "cloud_build.template.json" \
+         "${SUBS_FILE}" "${PROJECT_ID}" "${SVC_ACCT}" "${KEY_FILE_PATH}" "${WAIT_FOR_RESULT}"
+    exit "${BUILD_FAILED}"
+}
+
+function test_command() {
+    cp /home/airflow/gcs/data/githubctl ./githubctl
+    chmod u+x ./githubctl
+    pwd; ls -l    ./githubctl
+
+    git config --global user.name "TestRunnerBot"
+    git config --global user.email "testrunner@istio.io"
+
+    ./githubctl \
+    --token_file="$TOKEN_FILE" \
+    --op=dailyRelQual \
+    --hub="gcr.io/$GCR_STAGING_DEST" \
+    --gcs_path="$GCS_BUILD_PATH" \
+    --tag="$VERSION" \
+    --base_branch="$BRANCH"
+}
+
+function modify_values_command() {
+    # TODO: Merge these changes into istio/istio master and stop using this task
+    gsutil -q cp gs://istio-release-pipeline-data/release-tools/test-version/data/release/modify_values.sh .
+    chmod u+x modify_values.sh
+
+    echo "PIPELINE TYPE is $PIPELINE_TYPE"
+    if [ "$PIPELINE_TYPE" = "daily" ]; then
+        hub="gcr.io/$GCR_STAGING_DEST"
+    elif [ "$PIPELINE_TYPE" = "monthly" ]; then
+        hub="docker.io/istio"
+    fi
+    ./modify_values.sh -h "${hub}" -t "$VERSION" -p "gs://$GCS_BUILD_BUCKET/$GCS_STAGING_PATH" -v "$VERSION"
+}
+
+function gcr_tag_success() {
+  pwd; ls
+
+  gsutil ls "gs://$GCS_FULL_STAGING_PATH/docker/"             > docker_tars.txt
+  grep -Eo "docker\\/(([a-z]|[0-9]|-|_)*).tar.gz"               docker_tars.txt \
+      | sed -E "s/docker\\/(([a-z]|[0-9]|-|_)*).tar.gz/\\1/g" > docker_images.txt
+
+  #gcloud auth configure-docker  -q
+  while read -r docker_image; do
+    gcloud container images add-tag \
+    "gcr.io/$GCR_STAGING_DEST/${docker_image}:$VERSION" \
+    "gcr.io/$GCR_STAGING_DEST/${docker_image}:$BRANCH-latest-daily" --quiet;
+    #pull_source="gcr.io/$GCR_STAGING_DEST/${docker_image}:$VERSION"
+    #push_dest="  gcr.io/$GCR_STAGING_DEST/${docker_image}:latest_$BRANCH";
+    #docker pull $pull_source
+    #docker tag  $pull_source $push_dest
+    #docker push $push_dest
+  done < docker_images.txt
+
+  cat docker_tars.txt docker_images.txt
+  rm  docker_tars.txt docker_images.txt
+}
+
+function release_push_github_docker_template() {
```

Comment:
where are these functions called? I don't seem to find references in this PR?

Reply:
it is a git mv not sure why it shows up as removed/added. The functions are called from the python code